userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
+
+If you only have images without corresponding annotations, you can 
+use the annoation utility built into the image_classifier. An example
+of it's usage is shown below with the **PetImages** subfolder above.
+
+```python
+import turicreate as tc
+
+# Load images (Note: you can ignore 'Not a JPEG file' errors)
```

Comment:
But should you? Why are we instructing users to ignore these errors?

Reply:
At the very least, you can describe them as warnings (since they're not fatal, right?) and comment that this will skip the files that cannot be loaded.